airflow/executors/celery_executor.py

```diff
@@ -116,7 +116,7 @@ def _execute_in_fork(command_to_exec: CommandType) -> None:
         args.func(args)
         ret = 0
     except Exception as e:  # pylint: disable=broad-except
-        log.error("Failed to execute task %s.", str(e))
+        log.exception("Failed to execute task %s.", str(e))
```

Comment:
I assume the purpose of this change is to have trackback info in logs?

If there is other purpose(s) for this change, please clarify so we have full info. Thanks.

Reply:
Yes, this is only to get the stack trace, since otherwise there is very little to help troubleshooting - I ran with this patch to be able to find the issue with mixed case env vars at all.